
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Internal Server Error
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.559+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_Update.well-knownentity_500", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTUpdateWellKnownentity500 extends Exception {

    private java.lang.Object putUpdateWellKnownentity500;

    public PUTUpdateWellKnownentity500() {
        super();
    }

    public PUTUpdateWellKnownentity500(String message) {
        super(message);
    }

    public PUTUpdateWellKnownentity500(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTUpdateWellKnownentity500(String message, java.lang.Object putUpdateWellKnownentity500) {
        super(message);
        this.putUpdateWellKnownentity500 = putUpdateWellKnownentity500;
    }

    public PUTUpdateWellKnownentity500(String message, java.lang.Object putUpdateWellKnownentity500, java.lang.Throwable cause) {
        super(message, cause);
        this.putUpdateWellKnownentity500 = putUpdateWellKnownentity500;
    }

    public java.lang.Object getFaultInfo() {
        return this.putUpdateWellKnownentity500;
    }
}
